packages/components/src/internal/components/permissions/PermissionAssignments.tsx

```diff
@@ -420,10 +420,9 @@ export const PermissionAssignments: FC<PermissionAssignmentsProps> = memo(props
     }, [roles, rootPolicy, rootRolesToShow]);
 
     const panelTitle = useMemo(() => {
-        if (!isAppHomeFolder(selectedProject, moduleContext) && selectedProject)
-            return selectedProject.name + ' Permissions';
-        return 'Application Permissions';
-    }, [selectedProject, moduleContext]);
+        if (!selectedProject) return 'Application Permissions';
+        return (selectedProject.path === HOME_PATH ? HOME_TITLE : selectedProject.name) + ' Permissions';
```

Comment:
Is the change to not use "Application" as Home project title intentional?

Reply:
Yes, this is for consistency between the Projects Page and the Permissions Page